sdk/spring/CHANGELOG.md

```diff
@@ -1,49 +1,169 @@
 # Release History
 
 ## 4.0.0-beta.4 (Unreleased)
```

Comment:
> ## 4.0.0-beta.4 (Unreleased)

Do we need to update the title here?
from 4.0.0-beta.4 to 4.0 GA?

Reply:
not in this pr

Reply:
I will create another pr to change all the versions.